pkg/k8s/k8s.go

```diff
@@ -12,6 +12,8 @@ import (
 
 const (
 	kubernetesConfigFilePathEnvVariable = "KUBECONFIG"
```

Comment:
Tiny nit: Should this be constant be capitalized like the rest of the constants?

Reply:
This constant isn't used outside of the k8s package, so I prefer to keep it lower-cased (package private).

Reply:
Ahh, I totally forgot about go's package private/public visibility naming convention